### ISSUE TITLE
wsgi service is running as root

### ISSUE THREAD

**Issue by infa-ddeore** [rchakode/kube-opex-analytics#15]
pod logs show warning about wsgi being run as root, it need to be run as nonroot user, logs:

```
*** Starting uWSGI 2.0.18 (64bit) on [Sat Jul 13 07:39:24 2019] ***
compiled with version: 7.4.0 on 28 June 2019 21:50:41
os: Linux-4.14.106-97.85.amzn2.x86_64 #1 SMP Fri Mar 15 17:07:54 UTC 2019
nodename: kube-opex-analytics-5cbcd6d4df-w5cd8
machine: x86_64
clock source: unix
detected number of CPU cores: 8
current working directory: /koa
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** WARNING: you are running uWSGI without its master process manager ***
your memory page size is 4096 bytes
detected max file descriptor number: 65536

```

**Comment by rchakode** [comment]
Hi,
Thanks to report this issue. 
It will be considered for future releases, meanwhile any contributions to fixing it will be welcomed. Just make a PR :)

**Comment by hmfritz** [comment]
I'm having a similar issue, were you able to fix this?

**Comment by rchakode** [comment]
Hi @hmfritz,
Sorry for that, will take a look on this issue in the next couple of days.

**Comment by rchakode** [comment]
Dear @hmfritz @infa-ddeore ,
We've worked on a improvement to introduce security context so to allow to run Kubernetes Opex Analytics in non privileged mode.
Before releasing officially this feature, can you please take a minute to test it and check that it fix the issue on your side? 

To do so:
 * checkout the latest [Helm resources](https://github.com/rchakode/kube-opex-analytics/tree/master/helm)
 * update `Chart.yaml` file to set the version to `0.4.2-rc1`
 * update `values.yaml` file and set the configuration `securityContext.enabled`  to `true`
 * update your deployment

Thank you in advance for your feedback.
R.

**Comment by infa-ddeore** [comment]
this particular warning is gone now, I didn't verify the app works or not though. 
Why don't you keep non root user as default? 

there is another warning, not sure if there is a need to worry about it.

```
*** WARNING: you are running uWSGI without its master process manager ***
```

**Comment by rchakode** [comment]
Thank you for the feedback.
All those warnings (even the one related master process) are now fixed in version 0.4.2 just released.
